src/H5ACpublic.h

```diff
@@ -783,7 +779,4 @@ typedef struct H5AC_cache_image_config_t {
 
 //! <!-- [H5AC_cache_image_config_t_snip] -->
 
-#ifdef __cplusplus
-}
-#endif
```

Comment:
extern "C" is only needed to prevent C++ name mangling in functions. It's not needed here since there are no public API calls in either H5AC or H5C.